### DISCUSSION TITLE
Batch file for a defined action

### DISCUSSION THREAD

**Discussion by jesuskyste** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#335] [Q&A]
Hi !

Thanks for your awesome work !

I was juste wondering if there was a way to launch an action from a shortcut.
I've got a stream deck and would like to assign a shortcut or launch a batch file to turn on a device managed by Google Assistant, but not by IFTTT.
I'm pretty noob in code and developping, so if you can help me..

Thanks again and have a nice day :)

**Comment by Melvin-Abraham** [comment]
This feature is planned for the future possibly in the form of Routines.
